README.md

```diff
@@ -14,56 +14,66 @@ such as Libvirt, QEMU and KVM.
 
 > All the following commands are executed from the root path of the repository
 
-1. Declare environment vars with the SSH connection parameters. 
-These can be declared as input vars inside a `.tfvars` file too
-
-```bash
-export TF_VAR_SSH_HOST="XXX.XXX.XXX.XXX"
-export TF_VAR_SSH_USERNAME="yourUsername"
-export TF_VAR_SSH_PASSWORD="yourPassword"
-export TF_VAR_SSH_PRIVATE_KEY_PATH="~/.ssh/id_ed25519"
-```
-
-2. Execute some REQUIRED previous scripts to bootstrap the host
-
-> By the moment, only recent Ubuntu versions are supported. 
-> Feel free to extend the OS support pushing your code to this repository.
-
-```bash
-# Copy current SSH key into the target host
-echo ${TF_VAR_SSH_PASSWORD} | ssh-copy-id -f ${TF_VAR_SSH_USERNAME}@${TF_VAR_SSH_HOST}
-
-# Give execution permissions to the helper scripts
-chmod -R +x ./scripts
-
-# Connect to the host machine by SSH and install the dependencies using passwordless authentication 
-# (user with sudo privileges required) 
-ssh -i ${TF_VAR_SSH_PRIVATE_KEY_PATH} ${TF_VAR_SSH_USERNAME}@${TF_VAR_SSH_HOST} \
-  "sudo bash ./scripts/prepare-host-ubuntu.sh ${TF_VAR_SSH_USERNAME}"
-```
-
-3. Configure Terraform to store the state of your infrastructure
-
-> We have configured S3 backend by default thinking on bare metal solutions like S3-compatible APIs provided by 
-> solutions like TrueNAS, which cover the problem of storing the .tfstate using the same approach as major cloud providers
-
-```bash
-# Set the right parameters for your S3 storage
-nano backends/config.s3.tfbackend
-
-# Init the process configuring your backend
-terraform init -backend-config=backends/config.s3.tfbackend
-```
-
-4. Declare resources related to VMs to create.
-
-> Change the file called `data.tf`.
-> Several examples can be found inside (yes, for Kubernetes)
-
-5. Create your VMs.
-```bash
-terraform apply
-```
+1. Declare environment vars with the SSH connection parameters.
+   These can be declared as input vars inside a `.tfvars` file too
+
+   ```bash
+    export TF_VAR_SSH_HOST="XXX.XXX.XXX.XXX"
+    export TF_VAR_SSH_USERNAME="yourUsername"
+    export TF_VAR_SSH_PASSWORD="yourPassword"
+    export TF_VAR_SSH_PRIVATE_KEY_PATH="~/.ssh/id_ed25519"
+    ```
+
+2. Install some REQUIRED dependencies in local machine
+
+   > To build the ISOs we need to have installed `mkisofs`.
+
+   ```bash
+   sudo apt install mkisofs
+   ```
```

Comment:
When I was applying the terraform script I got an error indicating the that tools is missing when the base iso was been builded.

Sorry but now I cat’t paste the exactly error because I don’t have my computer with me. 

I you want I can test it again but I will be out of the office by two weeks.

Reply:
Yes, as this is something only happening to you AFAIK, please, I would like to have the output to analyze it before merging :)

Reply:
Ok, I'll do it.

Reply:
Hi again!!!

The error that appears if I haven't installed the tool, is the following:

```
module.workload.libvirt_volume.kube_disk["kube-loadbalancer-0"]: Creation complete after 0s [id=/opt/libvirt/metal-cloud-volume-pool/kube-disk-kube-loadbalancer-0.qcow2]
╷
│ Error: error while starting the creation of CloudInit's ISO image: exec: "mkisofs": executable file not found in $PATH
│ 
│   with module.workload.libvirt_cloudinit_disk.cloud_init["kube-loadbalancer-0"],
│   on modules/workload/storage.tf line 79, in resource "libvirt_cloudinit_disk" "cloud_init":
│   79: resource "libvirt_cloudinit_disk" "cloud_init" {
│ 
╵
```

I have Ubuntu 22.04 Desktop in my workstation, but using minimal installation, maybe that may affect it.

When I installed the tool, the error was gone.

Reply:
Hello there. Sorry for the delay but needed some time to check everything. It seems that [there is a discussion](https://github.com/dmacvicar/terraform-provider-libvirt/issues/465) regarding to this. Bu the moment, let's integrate this and follow the discussion there for the future